### ISSUE TITLE
Pybind11 build errors

### ISSUE THREAD

**Issue by yrdohr** [OxfordTechnicalSolutions/gad-sdk#25]
```

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(441,16): error C2065: 'trace': undeclared identifier [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(442,13): error C2065: 'trace': undeclared identifier [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(442,21): error C2065: 'trace': undeclared identifier [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(444,32): error C2065: 'trace': undeclared identifier [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(449,31): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(451,24): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(452,21): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\detail\type_caster_base.h(454,10): error C2059: syntax error: '/' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2150,49): error C2039: 'frame': is not a member of '_ts' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\cpython\pystate.h(82,8):
        see declaration of '_ts'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2151,36): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2152,9): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2155,13): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2155,30): error C2027: use of undefined type '_frame' [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\include\pytypedefs.h(22,16):
        see declaration of '_frame'

      C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2154,33): error C2660: 'pybind11::detail::dict_getitem': function does not take 1 arguments [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pytypes.h(511,19):
        see declaration of 'pybind11::detail::dict_getitem'
        C:\libs\gad-sdk\oxts-sdk-py\pybind11\include\pybind11\pybind11.h(2154,33):
        while trying to match the argument list '()'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(131,1): warning C4005: 'PRId8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(114,1):
        see previous definition of 'PRId8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(133,1): warning C4005: 'PRId32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(116,1):
        see previous definition of 'PRId32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(134,1): warning C4005: 'PRId64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(117,1):
        see previous definition of 'PRId64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(135,1): warning C4005: 'PRIu8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(171,1):
        see previous definition of 'PRIu8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(137,1): warning C4005: 'PRIu32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(173,1):
        see previous definition of 'PRIu32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(138,1): warning C4005: 'PRIu64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(174,1):
        see previous definition of 'PRIu64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(139,1): warning C4005: 'PRIo8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(152,1):
        see previous definition of 'PRIo8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(141,1): warning C4005: 'PRIo32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(154,1):
        see previous definition of 'PRIo32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(142,1): warning C4005: 'PRIo64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(155,1):
        see previous definition of 'PRIo64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(143,1): warning C4005: 'PRIx8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(190,1):
        see previous definition of 'PRIx8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(145,1): warning C4005: 'PRIx32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(192,1):
        see previous definition of 'PRIx32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(146,1): warning C4005: 'PRIx64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(193,1):
        see previous definition of 'PRIx64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(147,1): warning C4005: 'PRIX8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(209,1):
        see previous definition of 'PRIX8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(149,1): warning C4005: 'PRIX32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(211,1):
        see previous definition of 'PRIX32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(150,1): warning C4005: 'PRIX64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(212,1):
        see previous definition of 'PRIX64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(151,1): warning C4005: 'SCNd8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(235,1):
        see previous definition of 'SCNd8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(153,1): warning C4005: 'SCNd32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(237,1):
        see previous definition of 'SCNd32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(154,1): warning C4005: 'SCNd64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(238,1):
        see previous definition of 'SCNd64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(155,1): warning C4005: 'SCNu8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(292,1):
        see previous definition of 'SCNu8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(157,1): warning C4005: 'SCNu32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(294,1):
        see previous definition of 'SCNu32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(158,1): warning C4005: 'SCNu64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(295,1):
        see previous definition of 'SCNu64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(159,1): warning C4005: 'SCNo8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(273,1):
        see previous definition of 'SCNo8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(161,1): warning C4005: 'SCNo32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(275,1):
        see previous definition of 'SCNo32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(162,1): warning C4005: 'SCNo64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(276,1):
        see previous definition of 'SCNo64'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(163,1): warning C4005: 'SCNx8': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(311,1):
        see previous definition of 'SCNx8'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(165,1): warning C4005: 'SCNx32': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(313,1):
        see previous definition of 'SCNx32'

      C:\libs\gad-sdk\oxts-sdk-core\include\oxts\core\BasicTypes.h(166,1): warning C4005: 'SCNx64': macro redefinition [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')
        C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt\inttypes.h(314,1):
        see previous definition of 'SCNx64'

      C:\libs\gad-sdk\oxts-sdk-gal-cpp\include\oxts\gal-cpp\gad_encoders\gad_encoder_bin.hpp(57,36): warning C4267: 'argument': conversion from 'size_t' to 'const unsigned int', possible loss of data [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
        (compiling source file '../../../oxts-sdk-py/gal-py-bindings.cpp')

        Please define _WIN32_WINNT or _WIN32_WINDOWS appropriately. For example:
        - add -D_WIN32_WINNT=0x0601 to the compiler command line; or
        - add _WIN32_WINNT=0x0601 to your project's Preprocessor Definitions.
        Assuming _WIN32_WINNT=0x0601 (i.e. Windows 7 target).
      C:\libs\boost_1_83_0\boost/bind.hpp(41): message : The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior. [C:\libs\gad-sdk\build\temp.win-amd64-cpython-311\Release\oxts_sdk.vcxproj]
      Traceback (most recent call last):
        File "C:\Users\rforsberg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\rforsberg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\rforsberg\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 251, in build_wheel
          return _build_backend().build_wheel(wheel_directory, config_settings,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\build_meta.py", line 404, in build_wheel
          return self._build_with_temp_dir(
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\build_meta.py", line 389, in _build_with_temp_dir
          self.run_setup()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\build_meta.py", line 480, in run_setup
          super(_BuildMetaLegacyBackend, self).run_setup(setup_script=setup_script)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
          exec(code, locals())
        File "<string>", line 117, in <module>
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\__init__.py", line 103, in setup
          return distutils.core.setup(**attrs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
          dist.run_commands()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\dist.py", line 963, in run_command
          super().run_command(command)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\wheel\bdist_wheel.py", line 368, in run
          self.run_command("build")
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\dist.py", line 963, in run_command
          super().run_command(command)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\command\build.py", line 131, in run
          self.run_command(cmd_name)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 318, in run_command
          self.distribution.run_command(command)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\dist.py", line 963, in run_command
          super().run_command(command)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\command\build_ext.py", line 88, in run
          _build_ext.run(self)
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 345, in run
          self.build_extensions()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 467, in build_extensions
          self._build_extensions_serial()
        File "C:\Users\rforsberg\AppData\Local\Temp\pip-build-env-0pbo4kbj\overlay\Lib\site-packages\setuptools\_distutils\command\build_ext.py", line 493, in _build_extensions_serial
          self.build_extension(ext)
        File "<string>", line 110, in build_extension
        File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1776.0_x64__qbz5n2kfra8p0\Lib\subprocess.py", line 413, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['cmake', '--build', '.', '--config', 'Release']' returned non-zero exit status 1.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for oxts-sdk
Failed to build oxts-sdk
ERROR: Could not build wheels for oxts-sdk, which is required to install pyproject.toml-based projects

```

**Comment by yrdohr** [comment]
potential fix - updating the version of pybind11 included

**Comment by smcca-oxts** [comment]
Should now be fixed in Release v2.0.1
